products/knownprojects/sql/dcp_application.sql

```diff
@@ -203,7 +169,7 @@ SELECT DISTINCT
 
     dcp_projectbrief,
 
-    dcp_projectdescription,
+    -- dcp_projectdescription,
```

Comment:
What's up with this one?

Reply:
just noticed that too! it's [a ZAP column I added](https://github.com/NYCPlanning/data-engineering/pull/168/files#diff-afd801c435253a3a9eec4615111e15d379274569ffce04a0d863af73b7f8ff12R44) to the standardized export that KPDB now uses

this should be uncommented